src/lavinmq/mqtt/broker.cr

```diff
@@ -0,0 +1,49 @@
+module LavinMQ
+  module MQTT
+    class Broker
+
+      getter vhost, sessions
+      def initialize(@vhost : VHost)
+        @queues = Hash(String, Session).new
+        @sessions = Hash(String, Session).new
```

Comment:
Won't these two always be the same? Should be enough with `@sessions`?

Reply:
yep, rest from using vhost